api/src/main/java/org/apache/iceberg/catalog/Catalog.java

```diff
@@ -28,7 +29,7 @@
 /**
  * Top level Catalog APIs that supports table DDLs and namespace listing.
```

Comment:
Can you give a brief explanation on the rationale for leaving out the namespace/table listing methods?